md-commons/src/main/java/com/wipro/ats/bdre/util/OozieUtil.java

```diff
@@ -173,8 +175,32 @@ public void persistBeanList(List beans, boolean printOnly) {
             } catch (IllegalAccessException e) {
                 LOGGER.error(e);
                 throw new MetadataException(e);
-            }
+            }*/
+            persistBeanTryCatch(bean);
         }
+        /*LOGGER.info("propertiesToBeSaved=" + propertiesToBeSaved);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)